codex/stores/blockstore.nim

```diff
@@ -54,6 +54,13 @@ method listBlocks*(self: BlockStore, onBlock: OnBlock): Future[?!void] {.base.}
 
   raiseAssert("Not implemented!")
 
+method close*(self: Blockstore) {.base.} =
```

Comment:
We probably want `close` to be `async` as well.

Reply:
it's not a library, just an internal API. We don't have any async open, and we don't need async close **ATM**

Reply:
I had already made the change as suggested, but hadn't seen @Bulat-Ziganshin's comment until after I pushed the change some time later. It's self-contained in a `[wip]` commit so I can drop it, or we can keep it.